plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -1238,9 +1238,17 @@ function woocommerce_template_loop_category_link_close() {
 	function woocommerce_taxonomy_archive_description() {
 		if ( is_product_taxonomy() && 0 === absint( get_query_var( 'paged' ) ) ) {
 			$term = get_queried_object();
+			
+			/**
+			 * Filters the archive's raw description on taxonomy archives.
+			 *
+			 * @param string $term_description
+			 * @param string $term
+			 */
+			$term_description = apply_filters( 'woocommerce_taxonomy_archive_description_raw', $term->description, $term );
```

Comment:
What do you think of dropping `_raw` (and removing _'raw'_ from the description in the docblock)—we felt it might be a little crisper and more readable without, but I also understand why it's here. What do you think?